apps/web/ee/components/apiKeys/ApiKeyListContainer.tsx

```diff
@@ -22,65 +24,63 @@ function ApiKeyListContainer() {
   const [editModalOpen, setEditModalOpen] = useState(false);
   const [apiKeyToEdit, setApiKeyToEdit] = useState<(TApiKeys & { neverExpires?: boolean }) | null>(null);
   return (
-    <>
-      <div className="flex flex-col justify-between truncate pl-2 pr-1 sm:flex-row">
-        <div className="mt-9">
-          <h2 className="font-cal text-lg font-medium leading-6 text-gray-900">{t("api_keys")}</h2>
-          <p className="mt-1 mb-5 text-sm text-gray-500">{t("api_keys_subtitle")}</p>
-        </div>
-        <div className="mb-9 sm:self-center">
-          <Button StartIcon={PlusIcon} color="secondary" onClick={() => setNewApiKeyModal(true)}>
-            {t("generate_new_api_key")}
-          </Button>
-        </div>
-      </div>
-      <LicenseRequired>
-        <QueryCell
-          query={query}
-          success={({ data }) => (
-            <>
-              {data.length > 0 && (
-                <List className="pb-6">
-                  {data.map((item) => (
-                    <ApiKeyListItem
-                      key={item.id}
-                      apiKey={item}
-                      onEditApiKey={() => {
-                        setApiKeyToEdit(item);
-                        setEditModalOpen(true);
-                      }}
-                    />
-                  ))}
-                </List>
-              )}
+    <QueryCell
+      query={query}
+      customLoader={<SkeletonLoader />}
+      success={({ data }) => (
+        <div className="border-b border-gray-200 py-8 pl-2 pr-1">
+          <ShellSubHeading
```

Comment:
Also applied the standard Shells we already have to handle this type of sections for consistency